app/commands.py

```diff
@@ -1410,6 +1373,29 @@ async def rmpriv(ctx: Context) -> Optional[str]:
     await t.remove_privs(bits)
     return f"Updated {t}'s privileges."
 
+@command(Privileges.DEVELOPER, hidden=True)
```

Comment:
I think admins should be able to assign donator privileges, I don't see a benefit in restricting the access to developers.